test/scripts/aks-dynamic-ip-allocation.sh

```diff
@@ -32,6 +32,7 @@ az aks create \
     --resource-group $RESOURCE_GROUP \
     --name $MEMBER_CLUSTER_1 \
     --node-count $NODE_COUNT \
+    --node-vm-size Standard_A2_v2 \
```

Comment:
can AKS not find the VM automatically?  What if A2_v2 is either not available or deprecated in the future? What kinda of error will we get?

Reply:
Sometimes AKS chooses a ARM64 VM which cannot consume the docker images we build for linux/amd64. If the VM is not found we will run into create AKS cluster failures